### DISCUSSION TITLE
Select options "parameters"

### DISCUSSION THREAD

**Discussion by clinvest** [filamentphp/filament#10528] [Help] [answered]
Hi all,

I wonder if there is a way to have more parameters in a select field? more than "value" and "label". Is there a way to have a value, label and an id ? when I check a select option html I see data-id and data-value. Is the a way to have pass some information in these attributes? or another solution?

```
<div id="choices--dataentity_id-item-choice-2" class="choices__item choices__item--choice choices__item--selectable" role="option" data-choice="" data-id="2" data-value="8" data-select-text="" data-choice-selectable="">Duhamel</div>
```

For example
```
Select::make('technologies')
    ->options([
        ['value' => 'tailwind, 'id' => 7, 'label' =>  'Tailwind CSS'],
        ['value' => 'alpine, 'id' => 10, 'label' =>  'Alpine JS']
    ])
```

Thank you in advance,

**Comment by leandrocfe** [comment, marked as answer]
select is a key - value, I think you can't do it

**Reply by clinvest** [reply]
Ok thank you!
